server/cyf.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS energizers CASCADE;
+DROP TABLE IF EXISTS raitings CASCADE;
+
+CREATE TABLE energizers (
+    id           SERIAL PRIMARY KEY,
+    name         VARCHAR(50) NOT NULL,
+    energizer    VARCHAR NOT NULL
```

Comment:
I suggest using `description` instead of `energizer`

Reply:
Good suggestion thank you